src/main.ts

```diff
@@ -16,6 +16,8 @@ if (process.argv[2] === '-v') {
   process.exit(0);
 }
 
+const inputFile = process.args[2] == "-" ? "/dev/stdin" : process.args[2];
+
 const input = JSON.parse(fs.readFileSync(process.argv[2], 'utf-8'));
```

Comment:
I think you also need to to use `inputFile` here?

Also note that we use single quote for strings (and not double quotes)

Reply:
Let me fix that!

Reply:
Thanks!